build_tools/bazel_to_cmake/bazel_to_cmake.py

```diff
@@ -160,28 +162,47 @@ def convert_directory(directory_path, write_files, allow_partial_conversion,
   if not os.path.isfile(build_file_path):
     return Status.NO_BUILD_FILE
 
+  autogeneration_tag = f"Autogenerated by {repo_relpath(os.path.abspath(__file__))}"
+
+  header = "\n".join(["#" * 80] + [
```

Comment:
I spent a really silly amount of time fiddling with string formatting for this. If you don't like it, please phrase your complaint in the form of a suggested edit :stuck_out_tongue:

Reply:
Heh, I originally suggested without the full box of `#` characters to avoid that problem :P this LGTM

Reply:
Yeah I realized that a ways into said fiddling. It really seems like something that shouldn't be that hard to do, and I was annoyed I had to use f-strings *and* string methods *and* list comprehensions (the last technically not necessary, but calling ljust and appending a "#" on every string individually was a bit gross)